app/models/opentofu_worker.rb

```diff
@@ -49,6 +49,13 @@ def unit_environment_variables
     }
   end
 
+  def configure_service_worker_deployment(definition)
+    super
+
+    definition[:spec][:template][:spec][:containers].first[:volumeMounts] << {:name => "cert-path", :mountPath => "/root"}
```

Comment:
Does the container have access to `/root`?

Reply:
Using this path now which is accessible from terraform runner container `/opt/app-root/src/`
Writing certificate and key on path `/opt/app-root/src/config/cert`